nix/README.md

```diff
@@ -1,29 +1,35 @@
 ### Nix support
 
+### Quickstart
+
+- `sudo bash -c “mkdir /nix && chmod 755 /nix && chown -R $USERNAME /nix”`
+- Run the multi-user command from `https://nixos.org/download.html#nix-install-linux`
+- Enable flakes: `mkdir ~/.config/nix; echo "experimental-features = nix-command flakes" > ~/.config/nix.conf`
+- `cd s2n-tls`
 ### Devshell
 
 To enter a development shell with everything needed to build and test, run `nix develop` at the root of the project.
 
 There are some helper scripts in the environment to make building easier, but if you're familiar with Nix, note that these are 
 separate from the buildPhase, configurePhase and checkPhase.
+### Configure and build
+
+From inside the devShell: `configure; build`.
```

Comment:
Perhaps add a note about "this takes a very large time because it build _all_ of the libcryptos". Also why does it build all of the libcryptos?

Reply:
It's a first cut- we will likely separate out the groupings of libcryptos into separate named devShells.